src/modules/emote_menu/stores/emote-menu-view-store.js

```diff
@@ -40,6 +47,17 @@ function createCategory(id, provider, displayName, icon, categoryEmotes = []) {
   };
 }
 
+let categoryOrder = storage.get(EmoteCategoriesOrderStorageKey);
+
+function organizeCategories(categories) {
+  if (categoryOrder == null) {
+    categoryOrder = [];
```

Comment:
curious: why do we set categoryOrder to empty list here? maybe we could handle default values when we set and organizeCategories?

```
let categoryOrder = storage.get(EmoteCategoriesOrderStorageKey) || [];

function sortCategories(categories) {
  if (categoryOrder.length === 0) {
    return categories;
  }
  return categories.sort((a, b) => categoryOrder.indexOf(a.category.id) - categoryOrder.indexOf(b.category.id));
}
```